### ISSUE TITLE
Once the AuthKey flow is starting in config, it can't be backed out of

### ISSUE THREAD

**Issue by sargonas** [SpanPanel/span#25]
If you select "Auth Key" from the config flow UI, instead of "Manual", there is no method to back out of this flow and change to Manual... hard-exiting the UI flow leaves the config flow in a bad state, requiring a reload of the Integration.

Ideally, there should be a back-arrow type flow to head back to the Manual/Key flow menu.

Tagging @etmelvin as you PRd in this new flow originally, just to give you a heads-up and an opportunity to tackle it if you wanted to, however I was going to take it upon myself to take a crack at it over this holiday break this week, so no pressure, just wanted to make sure you were aware just in case.

**Comment by gdgib** [comment]
@sargonas I'm obviously triaging a bit here.  Is this still an issue with the recent changes I accepted from you?  If so, I'll work on it in the next few weeks for 0.0.9.

**Comment by sargonas** [comment]
Yeah, nothing I touched actually had an impact on fixing this. This was actually raised to me as an issue by someone else that surfaced along back here. I can replicate it myself as well, and I tried taking a stab at it, but admittedly the config flow UI stuff was a little bit of a learning curve for me at the time. Unfortunately I never went back to look at it again after.

**Comment by gdgib** [comment]
Thanks! I'll look into it, then for 0.0.9

**Comment by cayossarian** [comment]
Fixed as of v1.0.x with merges from SpanPanel/SpanCustom
